tests/ui/traits/bad-sized.stderr

```diff
@@ -9,33 +9,33 @@ LL |     let x: Vec<dyn Trait + Sized> = Vec::new();
    = help: consider creating a new trait with all of these as supertraits and using that trait here instead: `trait NewTrait: Trait + Sized {}`
    = note: auto-traits like `Send` and `Sync` are traits that have special properties; for more information on them, visit <https://doc.rust-lang.org/reference/special-types-and-traits.html#auto-traits>
 
-error[E0277]: the size for values of type `dyn Trait` cannot be known at compilation time
+error[E0277]: the size for values of type `dyn Trait<Metadata = ()>` cannot be known at compilation time
```

Comment:
This happens, because we do lower multiple principals in trait objects like `dyn Trait + Copy` during astconv (HIR->Ty lowering), but most code that consumes `Ty`s does not handle trait objects with multiple principals correctly, including the pretty printer.

I can fix this in a follow-up, either by making the pretty print work with multiple principals or by just stopping to lower more than one principal.